kano_profile_gui/badge_screen.py

```diff
@@ -217,6 +217,9 @@ def _create_info_box(self):
         description_label.set_line_wrap(True)
         description_label.set_line_wrap_mode(Pango.WrapMode.WORD)
```

Comment:
Might be an idea to change this to `Pango.WrapMode.WORD_CHAR` so that should a word be longer than 20 characters (which admittedly it probably won't be but there is a chance) then it will wrap.